cards-examples/Web/Web-card-article.md

```diff
@@ -10,229 +10,249 @@ path: /
-  * [Class definition](https://)
-  * [GitHub source](https://github.com/material-components/)
+You can also place cards within layout containers, such as [MDC Layout Grid](../mdc-layout-grid)
+or CSS Flexbox or Grid.
 
+### JavaScript
 
-The following example shows an elevated card. The card has a title, a secondary title, text, and two actions: Action 1 and Action 2 in purple (#6200EE).
+MDC Card itself does not require JavaScript. However, if you place interactive components inside your cards,
+you may want to instantiate ripples or other components. For example:
 
-<img src="assets/<platform>-elevated-card.png" alt="elevated card example for <platform> showing ...">
+```js
+import {MDCRipple} from '@material/ripple';
 
+const selector = '.mdc-button, .mdc-icon-button, .mdc-card__primary-action';
+const ripples = [].map.call(document.querySelectorAll(selector), function(el) {
+  return new MDCRipple(el);
+});
 ```
-<source code here>
-The source code example should display as per the interactive example (https://material.io/components/cards/#) with supporting text and Buttons:
-* Display two elevated cards, for each card
-* Display a card title "Card title 1" for one card and "Card title 2" for the other
-* Display a secondary title "Secondary text" with an opacity of 60%
-* Display text reading "Greyhound divisively hello coldly wonderfully marginally far upon excluding." with an opacity of 60%
-* Display two actions, "Action 1" and "Action 2" with two text buttons
-* Display the sample images ![sample card image of yellow and red tulips](assets/card-sample-image.jpg) and ![sample card image of red and yellow apples](assets/card-sample-image-2.jpg)
-* Allow the cards to be moveable.
-```
-
-### Key properties
-
-![Card anatomy diagram](assets/card-anatomy.png)
 
-**1. Elevated card attributes**
+> _NOTE_: If your card includes any [icon button toggles](../mdc-icon-button#icon-button-toggle), you will want to
+> instantiate `MDCIconButtonToggle` instead of `MDCRipple`.
 
-1. **Container** 
-2. **Thumbnail [optional]** 
-3. **Header text [optional]** 
-4. **Subhead [optional]** 
-5. **Media [optional]** 
-6. **Supporting text [optional]** 
-7. **Buttons [optional]** <
-8. **Icons [optional]** 
+## Card 
 
-<b>Container</b>
-
-Design Attribute | Theme value | Equivalent Sass mixin attribute
----|---|---
-Container fill color| |
-Container ink color | |
-Container shape radius | |
-Container outline color | |
-Container outline width | | 
-Container horzontal padding | | 
-Container elevation | |
+### Card example
 
-<b>Thumbnail (optional)</b>
+```html
+<div class="mdc-card">
+  <!-- ... content ... -->
+</div>
+```
 
+**Note**: MDC Card is designed to accommodate a wide variety of use cases. See the [Card Contents](#card-contents) section for information on helpers for specific types of content within cards.
 
-|  | Attribute | Related method(s) | Default value |
-|---|---|---|---|
-|Desc. 1 | | | |
+## Other Variants
```

Comment:
Use sentence case for titles: capitalize the first word, and only capitalize following proper nouns.